hip-service/wwwroot/swagger.yaml

```diff
@@ -350,7 +350,7 @@ components:
         transactionId:
           type: string
           format: uuid
-        link:
+        patient:
           type: object
           properties:
             careManagerUserID:
```

Comment:
What is careManagerUserId? Isn't this `consentManagerUserID`? Also rename `patientReferenceNumber` to 
`referenceNumber` as its already inside a patient object.